adapter/statsd2/statsd.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package statsd
+package statsd // import "istio.io/mixer/adapter/statsd"
```

Comment:
extra comment ?

Reply:
These comments are actually not comments, but directives to the compiler. See https://stackoverflow.com/questions/37330303/forcing-a-specific-import-path-in-go. Doug recommended we start using these.

It's ridiculous that this looks like a normal comment, but that's what the language designers chose.